documentation/testing-docs/test-matrix-for-az-accounts.md

```diff
@@ -0,0 +1,196 @@
+# Test Matrix for Authentication in Az.Accounts
+
+The authentication functionality in Az.Accounts is one of most important piece for Azure PowerShell, to make sure Azure PowerShell be delivered to customers with high quality, we define the test matrix and it should be honored by each release of Az.Accounts.
```

Comment:
The authentication functionality in Az.Accounts is one of the most important pieces in Azure PowerShell. To make sure Azure PowerShell be delivered to customers with high quality, we define the test matrix which should be honored by each release of Az.Accounts when there is any change related to authentication.